src/components/Pages/HomePage/Graphs.js

```diff
@@ -167,6 +170,7 @@ class Graphs extends React.Component {
             </article> */}
           </div>
         </div>
+        { impactPageEnabled ? (
```

Comment:
@BradHN1  did you mean `impactEnabled`?

Reply:
Thanks for catching that - it should have been impactPageEnabled but mistyped it above that.  Clearly should have checked it.  I've pushed a fix.